bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -88,9 +88,8 @@ export class PlotCanvasView extends BokehView
     @canvas_view.render(true)
 
     # If requested, try enabling webgl
-    if @model.plot.webgl or window.location.search.indexOf('webgl=1') > 0
-      if window.location.search.indexOf('webgl=0') == -1
-        @init_webgl()
```

Comment:
@bryevdv, I though this was a developers' tool, but apparently it's in the documentation.